src/EventConsumerService.h

```diff
@@ -1,11 +1,12 @@
-// Copyright (C) Sven Rosvall (sven@rosvall.ie)
+// Copyright (C) Martin Da Costa 2023 (martindc.merg@gmail.com)
 // This file is part of VLCB-Arduino project on https://github.com/SvenRosvall/VLCB-Arduino
```

Comment:
Sorry about that. I think I just copied the copyright notice when I shrunk the copyright notice you had here first. Thanks for fixing.

Reply:
No worries